stackslib/src/blockstack_cli.rs

```diff
@@ -314,8 +315,10 @@ fn sign_transaction_single_sig_standard(
     transaction: &str,
     secret_key: &StacksPrivateKey,
 ) -> Result<StacksTransaction, CliError> {
-    let transaction =
-        StacksTransaction::consensus_deserialize(&mut io::Cursor::new(&hex_bytes(transaction)?))?;
+    let transaction = StacksTransaction::consensus_deserialize_with_epoch(
```

Comment:
If you're going to use `StacksEpochId::latest()`, then you can just use `consensus_deserialize`